massa-pool-exports/src/controller_traits.rs

```diff
@@ -9,8 +9,11 @@ use massa_models::{
 };
 use massa_storage::Storage;
 
+#[cfg(feature = "test-exports")]
+use std::sync::{Arc, RwLock};
```

Comment:
unused ?

Reply:
They are used by mockall_wrap::wrap because my macro doesn't import them. The macro is a bit buggy but works as a first version I want to improve it when I will have all the tests re-written